plugin/build.gradle

```diff
@@ -218,9 +218,7 @@ List<String> jacocoExclusions = [
         'org.opensearch.ml.task.MLTaskCache',
         'org.opensearch.ml.task.MLTaskManager',
         'org.opensearch.ml.task.MLTrainAndPredictTaskRunner',
-        'org.opensearch.ml.rest.RestMLGetModelAction',
-        'org.opensearch.ml.rest.RestMLDeleteModelAction',
-        'org.opensearch.ml.rest.*'
+        'org.opensearch.ml.rest.AbstractMLSearchAction.1'
```

Comment:
Why add ".1"?

Reply:
If no this line, the error would be: 
> Rule violated for class org.opensearch.ml.rest.AbstractMLSearchAction.1: branches covered ratio is 0.0, but expected minimum is 0.8
  Rule violated for class org.opensearch.ml.rest.AbstractMLSearchAction.1: lines covered ratio is 0.0, but expected minimum is 0.3

I am checking where is that .1 coming from.

Reply:
It comes from the inner anonymous class defined in BaseHandler. There's a $1 class file generated from the build. I have replaced it with * to not confuse people.